jep/1/README.adoc

```diff
@@ -1120,11 +1173,35 @@ This JEP leverages existing infrastructure.
 
 * link:https://groups.google.com/d/msg/jenkinsci-dev/spDAr8EJm3c/T9Nmhn-fAQAJ[Request for feedback: Jenkins Enhancement Proposal]
 * link:https://groups.google.com/d/topic/jenkinsci-dev/tw0ETwvboAM/discussion[Modification of "Accepted" JEPs]
+* link:https://groups.google.com/g/jenkinsci-dev/c/hepntz6WZak[Changes in the JEP process and the BDFL role] (2020-2021)
 
 === Pull Requests
 
+Initial release:
+
 * link:https://github.com/jenkinsci/jep/pull/1[PR 1]
 * link:https://github.com/jenkinsci/jep/pull/12[PR 12]
 * link:https://github.com/jenkinsci/jep/pull/19[PR 19]
 * link:https://github.com/jenkinsci/jep/pull/76[PR 76]
 
+== Change history
+
+This section lists major changes in this process JEP.
+
+=== May TODO, 2021
+
+JEP-1 was modified according to the discussion in link:https://groups.google.com/g/jenkinsci-dev/c/hepntz6WZak[this thread] and the Jenkins governance meeting.
+Key changes:
```

Comment:
the tl;dr for those of us who are not lawyers

Reply:
```suggestion
The goal is to simplify the process by removing the BDFL delegation bottleneck and adopting the standard community decision making process instead.

Key changes:
```

Does it work for you @jglick ?

Reply:
Sure, looks fine